quickstart/configure-mayastor.md

```diff
@@ -15,12 +15,20 @@ A pool is defined declaratively, through the creation of a corresponding `DiskPo
 
 #### Permissible Schemes for `spec.disks` under DiskPool CR
 
+{% hint style="info" %}
+It is <b>highly recommended</b> to specify the disk using its `UUID_number` to ensure unique identification.
+The usage of `dev/sdx` might result in data corruption issues.
```

Comment:
dev/sdx --> /dev/sdx ??

Reply:
rather than `/dev/sdx` maybe say `device names (for example /dev/sda)`

Reply:
updated the text.